containers/deploy/puppet/tasks/main.yml

```diff
@@ -1,4 +1,11 @@
 ---
+- name: puppet persistent volume claim
+  k8s_raw:
+    state: "{{ deployment_state }}"
+    definition: "{{ lookup('template', 'templates/' + item + '.pvc.yaml') | from_yaml }}"
+  with_items:
+    - puppet-data
```

Comment:
This move means that Pulp now needs to require the Puppet role because Pulp mounts this volume. Which feels a bit odd to be honest, but that would at least keep it consistent for now.